src/main/java/org/zalando/nakadi/filters/LoggingFilter.java

```diff
@@ -243,4 +244,10 @@ private boolean isPublishingRequest(final RequestLogInfo requestLogInfo) {
                 requestLogInfo.path.startsWith("/event-types/") &&
                 (requestLogInfo.path.endsWith("/events") || requestLogInfo.path.endsWith("/events/"));
     }
+
```

Comment:
I guess you are going to use this in the next PR right?

Reply:
Yes